### PR TITLE
Blog migration prep

### PR DESCRIPTION
**Pull request by yathomasi**
- [x]  Create redirects on [[dvc.org](http://dvc.org/)](http://dvc.org) (302 temporary redirects)
- [x]  remove all blogs-related code 
- [x]  remove all the blogs content
- [x]  remove blog related logics
    - [x]  feed
- [x]  remove Github Action
- [x]  remove blog referenced (updated)
    - [x]  navbar
    - [x]  community page (used rss feed)
    - [x] WhatsNew section (used rss feed)
- [x]  Remove Subscribe from dvc (keep it as it for now) (Follow up after blog category/tabs)
- [x] Merge after (https://github.com/iterative/iterative.ai/pull/481) and update source site link https://github.com/iterative/dvc.org/blob/f56a73e79ad6f46ce7c960601dcd143ec91fb000/gatsby-config.js#L74-L75

### CODE REVIEW

**Comment by yathomasi** [comment]
I think the approach for merging this will be:
1. First merge the main PR on iterative.ai: https://github.com/iterative/iterative.ai/pull/481
2. Update the feed link https://github.com/iterative/dvc.org/blob/f56a73e79ad6f46ce7c960601dcd143ec91fb000/gatsby-config.js#L74-L75
3. Check on preview deployment and finally merge this PR.

File: src/components/Home/WhatsNewModal/index.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import * as styles from './styles.module.css'
 
 const WhatsNewModal: React.FC = () => {
   const {
-    allBlogPost: {
+    allFeedIterativeBlog: {
       nodes: [latestPost]
     }
   } = useStaticQuery(query)
```
Comment (rogermparent) [inline src/components/Home/WhatsNewModal/index.tsx:19]: Nice touch using the RSS feed for this!

**Review by rogermparent — APPROVED** [review]
Great work! This PR is a lot less intimidating to review when you skip over all the deleted files. I think the plan you've outlined moving forward is also solid.

**Comment by yathomasi** [comment]
Converting to draft and waiting for https://github.com/iterative/iterative.ai/pull/481 to get merged.

**Pull request merged**